README.md

```diff
@@ -6,4 +6,10 @@ Barefoot Nomad - Making company travel and accomodation easy and convinient.
 ## Vision
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
----
\ No newline at end of file
+---
+## Docker Development Setup
+
+- Install Docker
+- Change to application root directory
+- Build a docker image with the following command `docker build -t . storm-backend`
```

Comment:
change you documentation from `docker build -t . storm-backend` to `docker build -t storm-backend .` . and `docker-composer up` to `docker-compose up`

Reply:
Fixed